ui-admin/src/portal/publish/PortalEnvDiffView.tsx

```diff
@@ -230,7 +234,14 @@ export default function PortalEnvDiffView(
     </div>
     <div className="d-flex justify-content-center mt-2 pb-5">
       { userHasPermission(user, portal.id, 'publish') && <>
-        <Button variant="primary" onClick={() => applyChanges(selectedChanges)}>
+        <Button variant="primary" onClick={() => {
+          if (isEmpty(selectedChanges.configChanges) &&
+              isEmpty(selectedChanges.studyEnvChanges.filter(studyChange => !isEmpty(studyChange.configChanges)))) {
```

Comment:
we might want to exclude `enableFamilyLinkage` as a change that triggers the confirmation, since that's a setting that we expect to match across environments

Reply:
would we still want this safeguard in place to prevent people from accidentally turning it off in sandbox and inadvertently propagating to irb or live?